themes/codeforpoland/views/home.jade

```diff
@@ -2,30 +2,40 @@ extends layout
 
 block content
   #landing-hero(style="background-image:url(" + brigade.heroImage + ");background-position:bottom center;")
+    //Front end work needed for lines 6-7
```

Comment:
@therebelrobot Sorry about that, I didn't put the comment in and forgot to remove it.